index.html

```diff
@@ -1,37 +1,115 @@
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
         </ul>
+        <a href="#" class="story button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
+      <section class="featured grid">
+        <h2 class="featured">Professional Art Printing Data</h2>
+        <p  class="featured">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="featured flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <a href="#" class="featured button">See Project</a>
+      </section>
```

Comment:
- [ ] Kindly change all the section tag for each card to an `article or div` tag, this will make your project more semantic
- [ ] Please ensure your cards look like the Figma design, add appropriate spacing, padding and margin
- [ ] Please follow the mobile first approach for designing, currently your mobile looks like this ⬇️ 



https://user-images.githubusercontent.com/69638013/165211471-3a7507f0-83c8-42c0-bb34-e8eea2004c8a.mp4

Reply:
resolved